src/api/firebase.js

```diff
@@ -122,3 +123,24 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export function comparePurchaseUrgency(data) {
+	const inActiveItems = sortItems(
+		data.filter((item) => {
+			return getItemDaysSinceLastPurchase(item) > 60;
+		}),
+	);
+
+	const activeItems = sortItems(
+		data.filter((item) => {
+			return (
+				item.dateLastPurchased === null ||
+				getItemDaysUntilNextPurchase(item) < 60
+			);
+		}),
+	);
+
```

Comment:
I like how your team decided to put your logical operator inside a return statement, making the code easily readable.  I don't think I've implemented it this way before.